yacman/yacman.py

```diff
@@ -337,8 +337,7 @@ def select_config(config_filepath=None,
     if config_filepath:
         if not check_exist or os.path.isfile(config_filepath):
             return config_filepath
-        _LOGGER.error("Config file path isn't a file: {}".
-                      format(config_filepath))
+        _LOGGER.error("Config file path isn't a file: {}".format(config_filepath))
```

Comment:
are you intentionally changing the formatting away from respecting the line length limit?